gradle.properties

```diff
@@ -4,3 +4,7 @@ target = 1.8
 proguardCoreVersion = 7.0.1
 kotlinVersion = 1.3.72
 kotlinxMetadataVersion = 0.1.0
+
+org.gradle.vfs.watch=true
```

Comment:
How do these properties improve the build for local development?

Reply:
I see you already added some more explanation for them in the commit here, nice! https://github.com/Guardsquare/proguard/pull/131/commits/7db916113537eaca09795a14ac14750445e2ab53

Reply:
Yes, sorry for missing the explanation here. These properties speed up the build and introduce very little risk.
- File system watching will be enabled by default in Gradle 7
- Local build cache will be enabled by default in Gradle 8. 
   - It does a lot to speed up builds when switching branches or running with `clean`
   - There is a small overhead on every build invocation. We want to reduce the overhead before enabling by default.
- Parallel project execution works for most builds, but is still considered "experimental" since there's a slim chance you have configured your build in a strange way that causes things to break. 
  - We always suggest trying to enable it and only disabling if issues arise.

Reply:
Nice, thanks for the explanation!